### DISCUSSION TITLE
使用rclone挂载webdav无法访问阿里云盘的文件。

### DISCUSSION THREAD

**Discussion by wuhuqifeiQWER** [AlistGo/alist#2824] [Q&A] [answered]
如题，我用alist挂载了onedrive和阿里云盘两种存储策略，使用rclone挂载alist的webdav到本地以后，onedrive文件能正常读取，读取阿里盘里的文件时终端返回“ERROR : IO error: 403 Forbidden”。但使用raidrive挂载webdav的时候未出现错误，均能正常读取。求解决办法。

**Comment by qearlziper** [comment, marked as answer]
你把302重定向换成本地代理试试

**Reply by wuhuqifeiQWER** [reply]
已解决，谢谢！
